db/schema.rb

```diff
@@ -615,17 +616,29 @@
     t.datetime "updated_at",             null: false
   end
 
+  create_table "quote_likes", force: :cascade do |t|
+    t.integer  "user_id",    null: false
+    t.integer  "quote_id",   null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "quote_likes", ["quote_id"], name: "index_quote_likes_on_quote_id", using: :btree
+  add_index "quote_likes", ["user_id"], name: "index_quote_likes_on_user_id", using: :btree
+
   create_table "quotes", force: :cascade do |t|
-    t.integer  "anime_id"
-    t.text     "content"
+    t.integer  "media_id",                               null: false
+    t.text     "content",                                null: false
+    t.integer  "character_id",                           null: false
     t.string   "character_name", limit: 255
-    t.integer  "user_id"
+    t.integer  "user_id",                                null: false
     t.datetime "created_at",                             null: false
     t.datetime "updated_at",                             null: false
-    t.integer  "positive_votes",             default: 0, null: false
+    t.integer  "likes_count",                default: 0, null: false
+    t.string   "media_type",                             null: false
   end
 
-  add_index "quotes", ["anime_id"], name: "index_quotes_on_anime_id", using: :btree
+  add_index "quotes", ["media_id"], name: "index_quotes_on_media_id", using: :btree
```

Comment:
You probably wanna drop this index and add a compound index on `media_type, media_id` (and another on `character_id`)